test/language/computed-property-names/computed-property-names-to-name-side-effects-object.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    to name, accessor side effects 2
```

Comment:
to name, accessor side effects object literal

and the other one

to name, accessor side effects class literal